src/editorKit.ts

```diff
@@ -23,22 +23,22 @@ import type { ItemMessagePayload } from '@standardnotes/snjs'
  * with the appropriate callbacks.
  */
 export interface EditorKitDelegate {
-  insertRawText: (text: string) => void
+  insertRawText?: (text: string) => void
```

Comment:
If this is required why make it optional?

Reply:
This is required when `supportsFileSafe` is `true`

Reply:
Ah ok

Reply:
You could possibly make a separate interface then for FileSafe functions

Reply:
I wonder if we could use conditional types for this. Example, when `supportsFileSafe` is `true`, we should ask for the FileSafe functions. Will take a deeper look at this later.